tests/Tests.elm

```diff
@@ -8,24 +8,30 @@ import Expect
 import Test exposing (..)
 
 
-set : EverySet Int
+type alias MyInt =
```

Comment:
For the sake of having the test implementing a closer to reality, how about we change this to:

`type MyInt = MyInt Int`

and then on `pref` for example you'd change it to:

```diff
+pred : MyInt -> Bool
+pred (MyInt x ) =
```